index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="main-section">
+
+        <section class="toolbar">
+          <nav class="desktop">
+            <div class="all-item"><h1><a href="#">My Logo</a></h1></div>
+            <div class="desk-item">
+              <div><a href="#portfolio">Portfolio</a></div>
+              <div><a href="#about">About</a></div>
+              <div><a href="#contact">Contact</a></div>
+            </div>
+            <span><img src="image/hmburger.png" alt="humburger-icon" class="burger"></span>
+            </nav>
+        </section>
+      </header>
+
+    <main class="main-parts">
+        <section class="headline-section">
+          <h1>I'm Shahira <br> Glad to see <br>you!</h1>
+          <p>I’m a software developer! I can help you build a product , feature or website Look through<br> some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+            contact me.</p>
+          <P class="connect">LET'S CONNECT</P>
```

Comment:
- [ ] Kindy make sure your `LET'S CONNECT` has an `<a> or a `<button>` tag.This is because let's connect is a link.